package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@docknetwork/sdk",
-  "version": "1.0.2",
+  "version": "1.1.0",
```

Comment:
please update version as i pushed a node-types change that is version 1.1.0

Reply:
Done